plugins/tech-insights/src/components/FactDetails/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { FactDetails } from './FactDetails';
```

Comment:
remember to always run prettier on all changes, you can generally enable auto-formatting and automatic newlines at end of files in your editor

Reply:
But you probably didn't want this file here. You aren't exporting this from the package (which is what we use index files for). So you can probably just remove this file and instead internally depend on the FactDetails file directly